spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/prometheus/PrometheusProperties.java

```diff
@@ -70,14 +55,6 @@ public void setDescriptions(boolean descriptions) {
 		this.descriptions = descriptions;
 	}
 
-	public io.micrometer.prometheus.HistogramFlavor getHistogramFlavor() {
```

Comment:
I reverted the change duplicating the ConfigurationProperties class and instead made a `HistogramFlavor` enum here to avoid requiring the micrometer-registry-prometheus-simpleclient module. The config adapter handles mapping to the Micrometer enum.